.chainsaw.yaml

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/configuration-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Configuration
+metadata:
+  name: configuration
+spec:
+  timeouts:
```

Comment:
Maybe we can increase these timeouts so that GHAs can pass? Based on failed GHA, it looks like setting up mgmt cluster is taking more than 5 mins.

Reply:
Yeah might have increase the timeout to 10m. But the error was with capl-controller-manager not being ready. Just reran the test to see if the issue persists